src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -538,46 +538,48 @@ struct
 
   module Network = struct
     type t =
-      {logger: Logger.t; table: Transition_frontier.t Network_peer.Peer.Table.t}
+      { logger: Logger.t
+      ; ip_table: (Unix.Inet_addr.t, Transition_frontier.t) Hashtbl.t
+      ; peers: Network_peer.Peer.t Hash_set.t }
```

Comment:
I don't think we need the `peers` table. Can we remove it?

Reply:
it's used for `get_random_peers`

Reply:
But `get_random_peers` is used nowhere in the test code. So, we don't need the `peers` table.